utils/m0reportbug

```diff
@@ -139,6 +139,14 @@ main() {
         echo "OK$(_time $t)" >&2
     done
 
+    # Create directory and put only files created today into it.
```

Comment:
I think instead of today, the last recent will be more appropriate.
The manual run of m0reportbug after few days should be also able to capture last failures happened few days back.